src/components/interface/filter/index.jsx

```diff
@@ -1,69 +1,133 @@
-  state = {
-    anchorEl: null
-  };
+  constructor(props) {
+    super(props);
+    this.state = {
+      options: {},
+      filtered: {
+        academicLevel: [],
+        awardConsideration: [],
+        competitionLevel: [],
+        danceSize: [],
+        danceStyle: [],
+        school: []
+      }
+    };
+  }
 
-  handleClick = (event) => {
-    this.setState({ anchorEl: event.currentTarget });
-  };
+  componentDidMount() {
+    this.subscribe = db.collection('settings').onSnapshot((querySnapshot) => {
+      const options = {};
+      querySnapshot.forEach((doc) => {
+        options[doc.id] = Object.keys(doc.data());
+      });
+      this.setState({ options });
+    });
+  }
 
-  handleClose = () => {
-    this.setState({ anchorEl: null });
-  };
+  componentWillUnmount() {
+    this.subscribe();
+  }
+
+  handleFilterClearAll = () => this.setState({
+    filtered: {
+      academicLevel: [],
+      awardConsideration: [],
+      competitionLevel: [],
+      danceSize: [],
+      danceStyle: [],
+      school: []
+    }
+  })
+
+  handleFilterChecked = (e) => {
+    const { name, value, checked } = e.target;
+    const { handleFilters } = this.props;
+    const { filtered } = this.state;
 
-  handleMenuClose = () => {
-    this.setState({
-      anchorEl: null
+    this.setState((prevState) => {
+      const category = prevState.filtered[name];
+      const filteredValues = checked ? category.concat(value) : category.filter(v => v !== value);
+      return {
+        filtered: {
+          ...prevState.filtered,
+          [name]: filteredValues
+        }
+      };
     });
+
+    handleFilters(filtered);
   };
 
   render() {
-    const { anchorEl } = this.state;
-    const menuItems = [
+    const { filtered, options } = this.state;
+    const { awardConsideration } = filtered;
+    const { classes } = this.props;
+    const choices = [
       {
-        key: 1,
-        caption: 'Academic Level',
-        options: [{ value: 'test', label: 'test' }, { value: 'test', label: 'test' }]
+        checked: awardConsideration.includes('specialAward'),
+        name: 'awardConsideration',
+        value: 'specialAward',
+        label: (
+          <React.Fragment>
+            {'Special Award Only   '}
+            <LensIcon fontSize="inherit" color="primary" />
+          </React.Fragment>
+        )
       },
       {
-        key: 2,
-        caption: 'Level of Competition',
-        options: [{ value: 'test', label: 'test' }, { value: 'test', label: 'test' }]
-      },
-      {
-        key: 3,
-        caption: 'Dance Style',
-        options: [{ value: 'test', label: 'test' }, { value: 'test', label: 'test' }]
-      },
-      {
-        key: 4,
-        caption: 'Dance Size',
-        options: [{ value: 'test', label: 'test' }, { value: 'test', label: 'test' }]
+        checked: awardConsideration.includes('choreoAward'),
+        name: 'awardConsideration',
+        value: 'choreoAward',
+        label: (
```

Comment:
Looking at these two labels, they're really similar. I don't really like this idea of passing down a node or string. I'd lean towards taking these 2 similar React.Fragments and just implementing them at the base level class. We'll just pass in something like 'specialAward' or 'choreoAward' and let that class decide what the color or text should be.

Let's try to keep props as simple as possible.